IPG-Automotive-UK/react-ui
# Migrating flaky tests to Playwright

## Changes




This change migrates the flaky unit tests that have been occurring [here](https://github.com/IPG-Automotive-UK/react-ui/actions/runs/6906451322/job/18791512769) to run using Playwright.
![image](https://github.com/IPG-Automotive-UK/react-ui/assets/106325764/013b8f7a-8101-497f-8627-8cc166bf171b)

I removed the block of tests regarding the `onChange` checks under VehicleSelect. I recreated the tests using Playwright and placed them in a file in the same directory but in to a `.spec.tsx` file run when Playwright calls the tests. 


- Updated version of @playwright/test
- Recreated flaky tests in Playwright
- Removed flaky tests from the code

This PR is only to fix the flaky tests. More changes like this will come once a strategy for using Playwright in RUI has been implemented.

## Testing notes

- Checkout this branch
- Run `npm i -f`
- Run `npm run test` 
- Check that VehicleSelect tests for `onChange` are not ran under Unit tests
- Check that VehicleSelect tests are run under Playwright - this can be looked at by running `npx playwright show-report` after running `npx playwright test`


## Author checklist before assigning a reviewer
